### ISSUE TITLE
Sanitize serviceURL parameter

### ISSUE THREAD

**Issue by Harsh14901** [devclub-iitd/SingleSignOn#85]
The `serviceURL` parameter is currently vulnerable to XSS attacks. If you click [HERE](https://auth.devclub.in/user/login/?serviceURL=javascript:alert(document.cookie)) then after logging in you will see your CASI node session cookies being displayed in the alert box.

**Note:** The CASI tokens are `httpOnly` hence they are **immune to XSS attacks**.

**Comment by bicycleman15** [comment]
Has this been implemented ? @Harsh14901
Should I close this ?

**Comment by Harsh14901** [comment]
No, I haven't found a completely reliable solution for this yet. We could either add a regex check to the `serviceURL` parameter but I am not sure if that could also be circumvented by the attacker or not. (As we do in CTF challenges :P)
